packages/drawing-response/src/drawing-response/drawable-palette.jsx

```diff
@@ -4,6 +4,7 @@ import { InputContainer } from '@pie-lib/render-ui';
 import Select from '@material-ui/core/Select';
 import MenuItem from '@material-ui/core/MenuItem';
 import { withStyles } from '@material-ui/core/styles';
+import classnames from 'classnames';
```

Comment:
Just to check, it might already be implemented correctly: 
Always make sure that what new imports are used are also present in dependencies in package.json. Even though it works locally, the build won't work unless all the libraries used are explicitly declared in package.json.